doc/tutorials/card_game/drag_drop_cards_full.py

```diff
@@ -218,9 +216,9 @@ def pull_to_top(self, card):
         index = self.card_list.index(card)
         # Loop and pull all the other cards down towards the zero end
         for i in range(index, len(self.card_list) - 1):
-            self.card_list[i] = self.card_list[i + 1]
+            self.card_list.swap(i, i+1)
         # Put this card at the right-side/top/size of list
-        self.card_list[len(self.card_list) - 1] = card
+        # self.card_list[len(self.card_list) - 1] = card
```

Comment:
This code doesn't seem to be functionally equivalent?

Reply:
It is equivalent run the 12th step, this file had some other logic bug iirc.

Reply:
Right, I take that back. I'd remove lines 220, 221 then. Maybe add a comment on how the other one will float to the top.

Reply:
This should be addressed in https://github.com/Ibrahim2750mi/arcade/pull/6